tt_metal/common/test_tiles.hpp

```diff
@@ -10,50 +10,61 @@
 
 #include <cstdint>
 #include <vector>
+#include <optional>
+#include "tt_metal/common/constants.hpp"
 #include "tt_metal/common/assert.hpp"
 #include "tt_metal/third_party/tracy/public/tracy/Tracy.hpp"
 #include "math.hpp"
 
 enum TensorLayout {
     LIN_ROW_MAJOR = 0, // standard element-wise row-major
-    TILED32_SWIZZLED = 1, // row-major of tiles 32x32, each tile is row-major-swizzled
-    TILED32_4FACES = 2,  // rowm major of tiles 32x32, each tile is 4 faces, each face is row-major, faces are swizzled
+    TILED_SWIZZLED = 1, // row-major of tiles, each tile is row-major-swizzled
+    TILED_NFACES = 2,  // rowm major of tiles, each tile is N (N = 1, 2, or 4) faces, each face is row-major, faces are swizzled
```

Comment:
typo room major.

Separate question : What is meant by row major swizzled.

Reply:
Btw what is TILED_SWIZZLED, what does swizzled refer to here. @yugaoTT

Reply:
nice catch.
I think it just means the rows are stored in a zig-zag style.

Reply:
Thanks, btw are you in a rush to push this ? I think I am going to spend most of today reviewing your code. Asking because its already approved.

Reply:
I'll push once the pipelines are passing, probably in a few hours.

Reply:
ok